### PR TITLE
Delay tracking

### PR DESCRIPTION
**Pull request by mmphego**

### CODE REVIEW

**Comment by mmphego** [comment]
@brickZA  Branch ready to be merged

File: mkat_fpga_tests/test_cbf.py [hunk 1]
```diff
@@ -570,19 +572,20 @@ def plot_and_save(freqs, actual_data, expected_data, plot_filename,
         # TODO NM 2015-09-04: We are only checking one of the results here?
         # This structure needs a bit of unpacking :)
         Aqf.equals(np.min(actual_phases[0][0]), np.max(actual_phases[0][0]),
-            "Check if the phase-slope with delay = 0 is zero.")
```
Comment (brickZA) [inline mkat_fpga_tests/test_cbf.py]: Perhaps try and factor this to a for loop. You can derive how many clock cycles from the data, and if needed have a list of test descriptions that go with the test delays

**Comment by mmphego** [comment]
@brickZA  Unable to revert back to changes before the merge.

**Comment by mmphego** [comment]
@brickZA Branch can be merged now.

**Pull request merged**